webui/src/pages/repositories/repository/utils.js

```diff
@@ -0,0 +1,31 @@
+function getRepoStorageConfig(configs, repo) {
```

Comment:
I was thinking about moving this part to the pluginManager,
But ended up thinking it's not worth the complexity.